enclave-tls/README.md

```diff
@@ -128,3 +129,13 @@ In the early bootstrap of enclave-tls, the debug message is mute by default. In
 ## Occlum LibOS
 
 Please refer to [this guide](docs/run_enclave_tls_with_occlum.md) to run Enclave Tls with [Occlum](https://github.com/occlum/occlum) and [rune](https://github.com/alibaba/inclavare-containers/tree/master/rune).
+
+## Non-SGX Enviroment:
+
+In non-sgx enviroment, it's possible to show error message as below when running ./enclave-tls-client --attester sgx_ecdsa -i x -p x command. According to Intel DCAP's implementation, during ecdsa attestation verification, if the libsgx_urts library
```

Comment:
s/show error message/show the error messages/
s/running ./enclave-tls-client --attester sgx_ecdsa -i x -p x command/running the command `enclave-tls-client --attester sgx_ecdsa`/

Reply:
s/during ecdsa attestation verification/when calling to sgx_qv_get_quote_supplemental_data_size()/